demisto_sdk/commands/common/constants.py

```diff
@@ -1065,13 +1065,26 @@ class PB_Status:
 EXCLUDED_DISPLAY_NAME_WORDS = ['partner', 'community']
 MARKETPLACES = ['xsoar', 'marketplacev2']
 
+# From Version constants
+FILETYPE_TO_DEFAULT_FROMVERSION = {
+    FileType.JOB: '7.0.0',
+    FileType.LISTS: '6.5.0',
+    FileType.PRE_PROCESS_RULES: '7.0.0',
+    FileType.GENERIC_TYPE: '6.5.0',
+    FileType.GENERIC_FIELD: '6.5.0',
+    FileType.GENERIC_MODULE: '6.5.0',
+    FileType.GENERIC_DEFINITION: '6.5.0'
+}
+GENERAL_DEFAULT_FROMVERSION = '6.2.0'
 DEFAULT_CONTENT_ITEM_FROM_VERSION = '0.0.0'
 DEFAULT_CONTENT_ITEM_TO_VERSION = '99.99.99'
 MARKETPLACE_MIN_VERSION = '6.0.0'
-DEFAULT_JOB_FROM_VERSION = '6.5.0'
+DEFAULT_JOB_FROM_VERSION = FILETYPE_TO_DEFAULT_FROMVERSION.get(FileType.JOB)
```

Comment:
not urgent, but please consider replacing this constant with the `.get` call around the SDK (shoudn't be complicated)